Source/Platform/DefaultOpenTK/Backend/Audio/NativeAudioSource.cs

```diff
@@ -281,7 +283,6 @@ internal bool PerformStreaming()
 
 					// Initially play source
 					AL.SourcePlay(handle);
-					stateTemp = AL.GetSourceState(handle);
```

Comment:
I noticed that this is actually not needed because stateTemp is not used after this call.